File: FirebaseLogin/app/src/main/java/com/codelab/firebaselogin/MainActivity.java
package com.codelab.firebaselogin;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.codelab.firebaselogin.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private static ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: CameraXApp/app/src/main/java/io/codelab/cameraxapp/MainActivity.kt
package io.codelab.cameraxapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import io.codelab.cameraxapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        binding = ActivityMainBinding.inflate(layoutInflater)
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
    }
}